.github/scripts/notes.js

```diff
@@ -0,0 +1,26 @@
+module.exports = async ({ core, fetch }) => {
+  const { VERSION } = process.env
+  const slug = `version-${VERSION.replaceAll('.', '-')}`
+
+  try {
+    const res = await fetch('https://wordpress.org/documentation/wp-json/wp/v2/wordpress-versions?per_page=50')
+    const data = await res.json()
+
+    const release = data.find((tag) => tag.slug === slug)
+    if (!release) {
+      throw Error('Release not found')
+    }
```

Comment:
Couldn't this fail on `data.find` if the `await res.json()` didn't succeed?

Reply:
It could, jumping to the catch section.
There are many reasons to fail (HTTP 500?) before this assertion, actually.
All of these are going to raise an error, caught by the catch (hopefully).

Reply:
Oh, yes, forgot about the outer try/catch 👍